src/_scss/layouts/default/header/_logo.scss

```diff
@@ -42,19 +42,19 @@
             color: $color-white;
             font-size: rem(8);
             font-weight: $font-semibold;
-            left: rem(195);
+            left: rem(189);
             letter-spacing: 0.14rem;
             padding: rem(2) rem(4);
             position: absolute;
             text-align: center;
             top: rem(-2);
             @include border-radius(2px);
-            @include media(360px) {
-                left: rem(228);
+            @include media(359px) {
+                left: rem(223);
             }
             @include media($large-screen) {
                 font-size: rem(10);
-                left: rem(280);
+                left: rem(274);
```

Comment:
Please make sure you've also accounted for the "beta" label positioning on the home page (you may need to override in `pages/homepage/header/header.scss`

You can see the source of each CSS property if you run `npm run sass` and look at the Chrome devtools element inspector.

Reply:
![screen shot 2017-11-30 at 12 55 24 pm](https://user-images.githubusercontent.com/3594363/33446865-62e088be-d5cf-11e7-91f0-35c40328c91d.png)

Reply:
The last commit should fix the homepage 'beta' label